<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="VisiblecheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    font-family: 'Impact', fantasy;
    font-size: 15px;
    color: red;
}


</string>
   </property>
   <property name="text">
    <string>visible</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
  font-family: 'Impact', fantasy;
  font: 15px;
  padding: 0px 0px;
  margin: 10px 20px;
  position: relative;
  border: 2px solid white;
  border-radius: 15px;
  background: rgba(255, 0, 0, 125);
  color: white;
}

QPushButton:hover {
  background: rgba(255, 0, 0, 200);
  color: #fff;
}


QPushButton:pressed {
    background: rgba(161, 6, 6, 200);
}

QLabel {
  font-family: 'Impact', fantasy;
  border: 2px solid black;
  background: rgba(161, 6, 6, 200);
  color: white;
}


</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ProTimespinBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    font-family: 'Impact', fantasy;
    font-size: 15px;
    color: red;
}


</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="EndScorespinBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    font-family: 'Impact', fantasy;
    font-size: 15px;
    color: red;
}


</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: 'Impact', fantasy;
    font-size: 15px;
    color: red;
}


</string>
   </property>
   <property name="text">
    <string>Prop time interval</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: 'Impact', fantasy;
    font-size: 15px;
    color: red;
}
</string>
   </property>
   <property name="text">
    <string>Winner's score when game ends</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
